x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -408,3 +408,15 @@ export function clearSelectedJobIdFromUrl(url) {
     }
   }
 }
+
+export function resetMlJobUrl(tabId, url) {
```

Comment:
I don't see this being used anywhere. Unless I'm missing something.

Reply:
I originally did this in case we want to reset the window's url after navigating to the page, but since it's not needed, I have deleted this in https://github.com/elastic/kibana/pull/65778/commits/f3c8367a68b4abffa1dff9f3b9b7c456ecc1552a.